src/lib.rs

```diff
@@ -303,7 +346,16 @@ pub fn decode_config_buf<T: ?Sized + AsRef<[u8]>>(input: &T,
                                                   config: Config,
                                                   buffer: &mut Vec<u8>)
                                                   -> Result<(), DecodeError> {
-    let input_bytes = input.as_ref();
+    let mut input_copy;
+    let input_bytes = if config.strip_whitespace {
+        input_copy = Vec::<u8>::with_capacity(input.as_ref().len());
+        input_copy.extend(input.as_ref().iter().filter(|b| !b" \n\t\r\x0c".contains(b)));
```

Comment:
We could allocate less here if we only used an intermediate buffer of, say, 4KiB and repeatedly filled it with non-whitespace chars and decoded that into the output buffer. Again, not sure it's really worth the effort since most whitespacey base64 is probably PEM files that are (1) small and (2) read once on startup, and we're not closing the door on people implementing their own faster logic if they need it.